### ISSUE TITLE
Update crashes my site

### ISSUE THREAD

**Issue by itasterix** [templaza/astroid-framework#612]
I'm actually using:
- Framework Version: 3.0.4
- Template Version: 2.6.5

I want to update everything to the last version. When I do that my website crashes.
Want I have found is the Template update that causes crash.
Crash means that I see error 404 on every page of the website, included home.

Please any help or suggestion ?

**Comment by sonvnn** [comment]
Please send me your website includes ftp acc to sonlv@tenplaza.com. I will help you check it

**Comment by itasterix** [comment]
> Please send me your website includes ftp acc to [sonlv@tenplaza.com](mailto:sonlv@tenplaza.com). I will help you check it

thank you, just sent an email

**Comment by itasterix** [comment]
Received mail back for failure, correct email seems sonlv@templaza.com with m instead of n
Can you confirm?

**Comment by sonvnn** [comment]
> Received mail back for failure, correct email seems [sonlv@templaza.com](mailto:sonlv@templaza.com) with m instead of n Can you confirm?

@itasterix Yes, I am sorry. It's my wrong char on my mobile. My email is sonlv@templaza.com. Please resend me your website. 

Thank you,
Sonny

**Comment by itasterix** [comment]
Just sent email again.
Thank you so much.
I waiting for your answer.

**Comment by sonvnn** [comment]
This issue happen because "System - Regular Labs - Modules Anywhere". I tried to disable it and your site works normally.

**Comment by itasterix** [comment]
> This issue happen because "System - Regular Labs - Modules Anywhere". I tried to disable it and your site works normally.
That's incredible!
I tought it was a bug of astroid, but you are right.
I had version 7 of Modules Anywhere. Now update to version 8 and enabled again and it worked!
Thank you so much! Great help from you !!
